### PR TITLE
Db get product mod

### PR DESCRIPTION
**Pull request by raunak-ra**

### CODE REVIEW

File: src/OnlineRetailPortal.MongoDBStore/MongoProductStore.cs [hunk 1]
```diff
@@ -58,13 +58,30 @@ public async Task<GetProductStoreResponse> GetProductAsync(string productId)
         public async Task<GetProductsStoreResponse> GetProductsAsync(GetProductsStoreEntity request)
         {
             List<MongoEntity> mongoEntities;
+            var pageSize = request.PagingInfo.PageSize;
+            var pageNumber = request.PagingInfo.PageNumber;
+            var collection = _db.GetCollection<MongoEntity>(_collection);
+            var sortDefinition = Builders<MongoEntity>.Sort.Ascending(request.ProductSort.Type);
```
Comment (achirametavisca) [inline src/OnlineRetailPortal.MongoDBStore/MongoProductStore.cs]: remove this

File: src/OnlineRetailPortal.MongoDBStore/MongoProductStore.cs [hunk 2]
```diff
@@ -58,13 +58,30 @@ public async Task<GetProductStoreResponse> GetProductAsync(string productId)
...
+                    sortDefinition = Builders<MongoEntity>.Sort.Ascending(request.ProductSort.Type);
+                }
+                else
+                {
+                    sortDefinition = Builders<MongoEntity>.Sort.Descending(request.ProductSort.Type);
+                }
+
+
+                mongoEntities = await collection.Find(FilterDefinition<MongoEntity>.Empty)
+                                                .Skip(pageNumber - 1)
```
Comment (achirametavisca) [inline src/OnlineRetailPortal.MongoDBStore/MongoProductStore.cs]: this should be multiplied by pageSize

**Review by achirametavisca — CHANGES_REQUESTED** [review]

File: src/OnlineRetailPortal.MongoDBStore/MongoProductStore.cs (hunk 1 above)
Comment (raunak-ra) [reply src/OnlineRetailPortal.MongoDBStore/MongoProductStore.cs]: done

File: src/OnlineRetailPortal.MongoDBStore/MongoProductStore.cs (hunk 2 above)
Comment (raunak-ra) [reply src/OnlineRetailPortal.MongoDBStore/MongoProductStore.cs]: done

**Review by achirametavisca — APPROVED** [review]
Merge changes

**Pull request merged**